docs/source/examples/lora_finetune.rst

```diff
@@ -204,7 +206,7 @@ model without any wrappers or custom checkpoint conversion logic.
 
 .. note::
     Whenever loading weights with :code:`strict=False`, you should verify that any missing or extra keys in
-    the loaded :code:`state_dict` are as expected. torchtune's LoRA recipe does this by default via
+    the loaded :code:`state_dict` are as expected. torchtune's LoRA recipes do this by default via e.g.
```

Comment:
Is "e.g." needed here?

Reply:
Technically yes because we now have different validation checks in single device and distributed LoRA recipes cause of that FSDP state dict business, and I definitely do not wanna get into that here. So I could leave it out, but this "e.g." is actually doing a bit of heavy lifting tbh